### PR TITLE
Added Reviews for each city

### PR DESCRIPTION
**Pull request by lucasungaretti**
Got 3 reviews from tripadvisor for each city, added logic so it appears for correct city. Tested all cities and reviews appear correctly. Also switched layout so map is before reviews. Made minor UI changes.

### CODE REVIEW

File: travvy/src/components/AttractionsList.vue [hunk 1]
```diff
@@ -118,13 +105,82 @@
...
       </div>
       </div>
       <br>
+      <br>
+      <br>
+      <hr size="3" width="60%" color="#E7EDF7"> 
+
+<!-- all reviews taken from TripAdvisor-->
+<!-- the reviews of each city will dynamically display based on the chosen destination -->
+      <h1>Traveller Tips</h1>
```
Comment (lkdfount) [inline travvy/src/components/AttractionsList.vue:121]: These all look great Lucas. Do you think we should say something like "Reviews from Trip Advisor"?

File: travvy/src/components/AttractionsList.vue [hunk 2]
```diff
@@ -118,13 +105,82 @@
...
+          <h1> No reviews available </h1>
+      </div>
+
+      <!--<div v-if="$store.state.recommendedAttractions.length > 0">
+      <p>{{$store.state.recommendedAttractions[0].city}} felt pretty safe but if you are worried about COVID it may not be the best time to go, it is tough to get around without being around a lot of people.</p>
+      <h3>Lisa, 28 from Winnipeg, ⭐ ⭐ ⭐ ⭐</h3>
+      <p>I spent time at {{$store.state.recommendedAttractions[0].attraction_name}}, I'd totally recommend it. Great view and less people.</p>
+      </div>
+      <div v-else>
+       </div>-->
```
Comment (lkdfount) [inline travvy/src/components/AttractionsList.vue:182]: We can remove this code. It is unnecessary.

**Review by lkdfount — APPROVED** [review]

File: travvy/src/components/AttractionsList.vue (hunk 1 above)
Comment (lucasungaretti) [reply travvy/src/components/AttractionsList.vue:121]: Just have it commented but I dont think its necessary

File: travvy/src/components/AttractionsList.vue (hunk 2 above)
Comment (lucasungaretti) [reply travvy/src/components/AttractionsList.vue:182]: i just wanted to keep it as a record if we wanted to do something with $store later

File: travvy/src/components/AttractionsList.vue [hunk 3]
```diff
@@ -118,13 +105,82 @@
...
+      
   </div>
 </template>
```
Comment (emilynairnn) [inline travvy/src/components/AttractionsList.vue:186]: Looks great Lucas! Maybe in a future sprint we could look into how this information could retrieved through an API!

**Review by emilynairnn — APPROVED** [review]

**Pull request merged**